lldb/source/Target/StackFrameList.cpp

```diff
@@ -979,6 +988,8 @@ size_t StackFrameList::GetStatus(Stream &strm, uint32_t first_frame,
     ++num_frames_displayed;
   }
 
+  if (filtered)
+    strm << "Note: Some frames were hidden by frame recognizers\n";
```

Comment:
Are we exposing the term "frame recognizers" to users intentionally?  Could we express this as "Language implementation stack frames hidden"?  I'm not convinced this is better, just asking, a lot of our errors and warning messages use lldb-internal terminology and I don't think it's helpful.

Reply:
There is a `frame recognizer` multiword command so I think it's fine to expose it to the user.

Reply:
Yes, this is intentional. Frame recognizers are visible to users and can (in theory) be extended via scripting:

```
(lldb) help frame recognizer
Commands for editing and viewing frame recognizers.

Syntax: frame recognizer [<sub-command-options>] 

The following subcommands are supported:

      add    -- Add a new frame recognizer.
      clear  -- Delete all frame recognizers.
      delete -- Delete an existing frame recognizer by id.
      info   -- Show which frame recognizer is applied a stack frame (if any).
      list   -- Show a list of active frame recognizers.

For more help on any particular subcommand, type 'help <command> <subcommand>'.
```